### ISSUE TITLE
Feature : Adding Group Description while creating the new Group

### ISSUE THREAD

**Issue by Asrani-Aman** [CircuitVerse/CircuitVerse#4765]
### Is your feature request related to a problem? Please describe

When creating a new group in CircuitVerse, users are currently only prompted to provide a group name. However, in educational settings, particularly for teachers who use CircuitVerse for class activities and assignments, additional information such as section/room designation or any description is often necessary for organisation and management purposes.

### Describe the solution you'd like

I propose adding a group description feature when creating a new group in CircuitVerse. This would allow users, especially educators and teachers, to provide additional context and information about the group, such as section or room details, class objectives, or any other relevant information.

Furthermore, I have created a [Figma design](https://www.figma.com/file/K2VtkUdgXvqMIUn3XRCXAd/Description-Box?type=design&node-id=0%3A1&mode=design&t=HiLXYTGwxBX8hZdp-1) to illustrate this proposed solution.

### Are you working on this?

Yes

**Comment by tanmoysrt** [comment]
Hi @Asrani-Aman , you need to do some extra also for this feature
- Add description while creating group
- Show group description in groups list
- In `edit group` page, provide support to update description as well

**Comment by Asrani-Aman** [comment]
yes @tanmoysrt bhaiya , i also have created a new revamp page for the group
